target/linux/ramips/image/mt7621.mk

```diff
@@ -506,6 +506,30 @@ define Device/netgear_r6850
 endef
 TARGET_DEVICES += netgear_r6850
 
+define Device/netgear_r6700v2
```

Comment:
Please use the shared definition netgear_r6260_r6350_r6850 here.

While thinking about it, it makes sense to shorten the name for this one and also use it for netgear_r6220. I will send a PR for that, which you might rebase on if you like (or not, if you prefer to stay independent).

Reply:
See #2510